core/src/main/java/com/netflix/conductor/core/events/ActionProcessor.java

```diff
@@ -48,158 +48,146 @@
 @Singleton
 public class ActionProcessor {
 
-	private static Logger logger = LoggerFactory.getLogger(ActionProcessor.class);
```

Comment:
It's nice to format in a separate commit, otherwise, we have to use external tools to see differences

Reply:
Yes, stop formatting existing code in a feature or bug fix PR, it is super hard to review PRs with these.